.env.ci

```
@@ -0,0 +1,31 @@
+# CI environment — encrypt with: dotenvx encrypt -f .env.ci
+# Then commit the encrypted version. Keep .env.keys safe (never commit it).
+#
+# These are minimal values for CI test runs (testcontainers provides DB/Valkey).
+
+# Server
+RAVEN_SERVER_PORT="8080"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)